opensciencegrid/default-build-config.json

```diff
@@ -0,0 +1,8 @@
+{
+    "base_os": {
+      "base_name": ["el7, el9"],
```

Comment:
I don't think we actually want a nested hash here. You should just use the same names as the ones we pass through to the build / push actions:

```
{
  "base_os": ["el7, el9"],
  "osg_series": ["3.6, 23"],
  "base_repo": ["development, "testing", "release"]
}
```

But how will you handle the next phase of the project? What will the configs look like to determine whether this is a trigger or an in-repo build?